_search-plugins/sql/functions.md

```diff
@@ -18,166 +18,170 @@ The SQL plugin supports the following common functions shared across the SQL and
+| locate    | `locate(string, string, integer) -> integer` or `locate(string, string) -> integer` | `SELECT locate('o', 'hello')`, `SELECT locate('l', 'hello', 3)` |
+| replace   | `replace(string, string, string) -> string`                                         | `SELECT replace('hello', 'l', 'x')`                             |
+| right     | `right(string, integer) -> string`                                                  | `SELECT right('hello', 1)`                                      |
+| rtrim     | `rtrim(string) -> string`                                                           | `SELECT rtrim('hello   ')`                                      |
+| substring | `substring(string, integer, integer) -> string`                                     | `SELECT substring('hello, 2, 4)`                                |
+| trim      | `trim(string) -> string`                                                            | `SELECT trim('   hello')`                                       |
+| upper     | `upper(string) -> string`                                                           | `SELECT upper('hello world')`                                   |
 
 ## Aggregate
 
-Function | Specification | Example
-:--- | :--- | :---
-avg | `avg(number T) -> T` | `SELECT avg(2, 3) FROM my-index LIMIT 1`
-count | `count(number T) -> T` | `SELECT count(date) FROM my-index LIMIT 1`
-min | `min(number T, number) -> T` | `SELECT min(2, 3) FROM my-index LIMIT 1`
-show | `show(string T) -> T` | `SHOW TABLES LIKE my-index`
+| Function | Specification            | Example                            |
+|:---------|:-------------------------|:-----------------------------------|
+| avg      | `avg(number T) -> T`     | `SELECT avg(column) FROM my-index` |
+| count    | `count(number T) -> T`   | `SELECT count(date) FROM my-index` |
+| min      | `min(number T) -> T`     | `SELECT min(column) FROM my-index` |
+| show     | `show(string) -> string` | `SHOW TABLES LIKE my-index`        |
 
 ## Advanced
 
-Function | Specification | Example
-:--- | :--- | :---
-if | `if(boolean, es_type, es_type) -> es_type` | `SELECT if(false, 0, 1) FROM my-index LIMIT 1`, `SELECT if(true, 0, 1) FROM my-index LIMIT 1`
-ifnull | `ifnull(es_type, es_type) -> es_type` | `SELECT ifnull('hello', 1) FROM my-index LIMIT 1`, `SELECT ifnull(null, 1) FROM my-index LIMIT 1`
-isnull | `isnull(es_type) -> integer` | `SELECT isnull(null) FROM my-index LIMIT 1`, `SELECT isnull(1) FROM my-index LIMIT 1`
+| Function | Specification                              | Example                                               |
+|:---------|:-------------------------------------------|:------------------------------------------------------|
+| if       | `if(boolean, es_type, es_type) -> es_type` | `SELECT if(false, 0, 1)`, `SELECT if(true, 0, 1)`     |
```

Comment:
can you double-check that these work please? 

- [ ] test

Reply:
I checked and there was one incorrect example, fixed now.

I am also realizing the it says es_type which I am assuming stands for elastic search type, I can change that to os_type